projects/scripts/adi_board.tcl

```diff
@@ -1131,6 +1173,7 @@ proc ad_cpu_interconnect {p_address p_name {p_intf_name {}}} {
           set p_address [expr ($p_address + 0x20000000)]
         }
       }
+      puts "create_bd_addr_seg -range $p_seg_range -offset $p_address $sys_addr_cntrl_space $p_seg_name \"SEG_data_${p_name}\""
```

Comment:
... and we can remove the escaped double quotes and curly braces.

Reply:
they aren't required I just added `puts` around the statement that was there. I'll fix it.

Reply:
I guess the only instance where it makes sense to keep it is if `$p_name` includes a space, but I'm not sure if that breaks other things too. Anyway, it's just a debug trace.